### DISCUSSION TITLE
What's a good way to limit bandwidth for a filebrowser installation with Docker?

### DISCUSSION THREAD

**Discussion by FunctionDJ** [filebrowser/filebrowser#3311] [Q&A]
Hi, i know about Linux `tc` or traffic shaping but i'm wondering what a good method of limiting the bandwidth for a filebrowser Docker container is.
I'd like to keep some amount of my upload bandwidth reserved for other services and applications (e.g. gaming).
For example, i'd like to limit the upload speed of the entire filebrowser app/container to 20mbit/s.

Do you guys have any ideas or recommendations?

**Comment by Rapter001** [comment]
use nginx reverse proxy
